### ISSUE TITLE
Allow that administrators submit revisions

### ISSUE THREAD

**Issue by t-wissmann** [CCSDForge/episciences#142]
On the "paper" page, i.e. pages with the url
```
https://lmcs.episciences.org/paper/view?id=DOCID
```
it should be possible for administrators, to submit a new version of the article. Right now, only the contributor can submit a new version, and so we at LMCS need to switch to the contributor's account and submit the new version.

A couple of years ago, this requested feature did already exist. I would assume that the implementation of this feature request simply amounts to the following change:
```diff
diff --git application/modules/journal/controllers/PaperController.php application/modules/journal/controllers/PaperController.php
index 1640a6d..feb75ab 100644
--- application/modules/journal/controllers/PaperController.php
+++ application/modules/journal/controllers/PaperController.php
@@ -334,7 +334,7 @@ class PaperController extends PaperDefaultController
         $this->view->copyEditingDemands = $copyEditingDemands;
 
         // reply copy editing answer form
-        if (Episciences_Auth::getUid() === $paper->getUid()) {
+        if (Episciences_Auth::isAdministrator() || Episciences_Auth::getUid() === $paper->getUid()) {
             $copyEditingReplyForms = Episciences_CommentsManager::getCopyEditingReplyForms($copyEditingDemands, $paper);
             $this->view->copyEditingReplyForms = $copyEditingReplyForms;
         }
```

**Comment by rtournoy** [comment]
It can be tested on https://epijinfo.episciences.org ; if it's ok it can be moved into production

**Comment by a3nm** [comment]
Hi, after some effort I found an article on epijinfo which is from arXiv and for which a more recent arXiv version exists: https://epijinfo.episciences.org/administratepaper/view?id=649

However, I do not see how, as an administrator, I can import the latest arXiv version of this paper. How is the new feature supposed to work?

**Comment by a3nm** [comment]
Ah, sorry, the article needs to have the correct status. Let me test further.

**Comment by a3nm** [comment]
Hi, OK, after some more testing, I think it works fine.

It is not very intuitive that the interface to submit a new version is on https://epijinfo.episciences.org/paper/view?id=1452 but not visible on https://epijinfo.episciences.org/administratepaper/view?id=1452 -- but I think this is good enough for our purposes and much better than the previous system.

Also, it is a bit strange that the new interface makes it possible to submit the "final version" multiple times, see https://epijinfo.episciences.org/administratepaper/view/id/649 "test multiple submissions". This is just FYI, it is not a problem for us as we do not use that feature.

Long story short, would it be possible to deploy this new feature to production? It will make our process more intuitive. Thanks!

**Comment by TobiasKappe** [comment]
Hi - can I ask that this feature be deployed to production? It'd help speed up our work as layout editors a lot! Thanks.

**Comment by rtournoy** [comment]
Hi, sorry for the delay, it has been fixed in another branch of code which needs more tests, although we should be able to push that online next week

**Comment by rtournoy** [comment]
Apologies for the delay, it's online
